sdk/core/Azure.Core/src/Shared/AppConfigHelper.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+#nullable enable
+
+namespace Azure.Core
+{
+    /// <summary>
+    /// Helper for interacting with AppConfig settings and their related Environment variable settings.
+    /// </summary>
+    internal static class AppConfigHelper
+    {
+        /// <summary>
+        /// Determines if either an AppContext switch or its corresponding Environment Variable is set
+        /// </summary>
+        /// <param name="appContexSwitchName">Name of the AppContext switch.</param>
+        /// <param name="environmentVariableName">Name of the Environment variable.</param>
+        /// <returns>If the AppContext switch has been set, returns the value of the switch.
+        /// If the AppContext switch has not been set, returns the value of the environment variable.
+        /// False if neither is set.
+        /// </returns>
+        public static bool GetConfigValue(string appContexSwitchName, string environmentVariableName)
```

Comment:
GetSwitchValue?

Reply:
I was thinking this name makes sense since we check both the switch value and the environment variable